test/api/unit_roles_test.rb

```diff
@@ -17,6 +17,22 @@ def test_get_unit_roles
     # asserts
   end
 
+  # Get a unit role's details
+  def test_get_a_unit_roles_details
+    expected_ur = UnitRole.second
```

Comment:
Same here - in this case build a unit and get the unit role from the unit itself.
You can then access this using the auth token of the unit's main convenor.